packages/titus-frontend/public/index.html

```diff
@@ -30,16 +30,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      rel="stylesheet"
+      href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
```

Comment:
this additional font is required by @material-ui, which is a peerDependency of brokeneck-react, for the time being